packages/misc/src/lib/misc.ts

```diff
@@ -204,14 +202,13 @@ export const checkType = ({
  * @returns { boolean }
```

Comment:
In the comments above:

`* @param { object } authSig` -> `* @param { JsonAuthSig } authSig`